### DISCUSSION TITLE
Live Update documentation fix

### DISCUSSION THREAD

**Discussion by alexookah** [capawesome-team/capacitor-plugins#204] [General]
Hello,

I noticed that the documentation in liveUpdate has something missing.
I was wondering whats the difference reset vs reload. So I noticed the reset and setBundle is not correct. Both have the same reload text. 

https://github.com/capawesome-team/capacitor-plugins/blob/a0627eb8e7191ab626c79c9993dc7638afca0a30/packages/live-update/src/definitions.ts#L139-L158

Can you have a look @robingenz  ?

**Comment by robingenz** [comment]
Hi @alexookah, thank you for your request. 

I assume you are referring to the following sentence:

> Call `reload()` or restart the app to apply the changes.

This sentence is correct and is required for both `reset()` and the `setBundle(...)` method. The `reset()` method resets the app to the initial bundle that was delivered with the native version. However, this "new" bundle is not applied immediately, but only after the next app start. If you want to apply the "new" bundle immediately, you can use the `reload(...)` method. The same applies to the `setBundle(...)` method.
